build/scripts/Unstub.ps1

```diff
@@ -25,4 +29,31 @@ $packageReferenceNode.SetAttribute("Version", "10.0.25148.1001-220626-1600.rs-fu
 $itemGroupNode = $xml.CreateElement("ItemGroup")
 $itemGroupNode.AppendChild($packageReferenceNode)
 $xml.DocumentElement.AppendChild($itemGroupNode)
-$xml.Save($projFile)
\ No newline at end of file
+$xml.Save($projFile)
```

Comment:
This is a dupe of the file in another location?  Can this file not be shared, or referenced?

If not, please add a comment to this file, and the original, stating that these two files need to stay in-sync.

Reply:
what do you mean this is a dupe?  this is the one and only file with the name "unstub" that I can find in my repo (and I'm sync'd up on main)